src/custom/components/CowSubsidyModal/constants.ts

```diff
@@ -3,15 +3,16 @@ export const SUBSIDY_INFO_MESSAGE = '(v)COW token holders are eligible for a fee
 export const COW_SUBSIDY_DATA: [string, number][] = [
   ['0', 0],
 
-  // 1,000
-  ['1000000000000000000000', 5],
-
-  // 10,000
-  ['10000000000000000000000', 10],
-
-  // 100,000
-  ['100000000000000000000000', 20],
-
-  // 1,000,000
-  ['1000000000000000000000000', 40],
+  // TODO: If we ever have subsidies again, make sure to update them here
+  // // 1,000
+  // ['1000000000000000000000', 5],
+  //
+  // // 10,000
+  // ['10000000000000000000000', 10],
+  //
+  // // 100,000
+  // ['100000000000000000000000', 20],
+  //
+  // // 1,000,000
+  // ['1000000000000000000000000', 40],
```

Comment:
This change effectively disables the subsidies, since all tiers have `0` discount.

No need for further UI changes besides removing the subsidy UI and modal